templates/Submissions/customize.php

```diff
@@ -221,7 +233,7 @@
                     </td>
 
                     <?php
-                    if ($submission->submission->equation && $valid) {
+                    if ($equation && $valid) {
```

Comment:
I was hoping to get some insight into how the custom equations operate, but it doesn't seem like anything in this patch is changing the functionality, only the descriptive text.  Was the support for those extra operators always there, and they just needed to be used?

Reply:
You can have an insight here: https://github.com/IO500/webpage/blob/main/src/Controller/SubmissionsController.php#L622

We are using the MathExecutor library (https://github.com/neonxp/MathExecutor/) to handle all the operations.